dataland-frontend/src/components/resources/dataTable/MultiLayerDataTable.vue

```diff
@@ -42,15 +55,86 @@
 }
 </style>
 
-<script setup lang="ts" generic="T">
+<script lang="ts" generic="T">
+import { defineComponent } from "vue";
 import { type MLDTConfig } from "@/components/resources/dataTable/MultiLayerDataTableConfiguration";
 import MultiLayerDataTableBody from "@/components/resources/dataTable/MultiLayerDataTableBody.vue";
 import { type DataAndMetaInformation } from "@/api-models/DataAndMetaInformation";
+import Tooltip from "primevue/tooltip";
+import { convertUnixTimeInMsToDateString } from "@/utils/DataFormatUtils";
+import {
+  type CompanyReport,
+  type EuTaxonomyDataForFinancials,
+  type EutaxonomyNonFinancialsData,
+  type SfdrData,
+} from "@clients/backend";
 
-defineProps<{
-  config: MLDTConfig<T>;
-  dataAndMetaInfo: Array<DataAndMetaInformation<T>>;
-  ariaLabel?: string;
-  inReviewMode: boolean;
-}>();
+export default defineComponent({
+  components: {
+    MultiLayerDataTableBody,
+  },
+  directives: {
+    tooltip: Tooltip,
+  },
+  props: {
+    config: {
+      // @ts-ignore
```

Comment:
Warum so viel TS-Ignore?

Reply:
@MarcTM01 TS seems to think Generic T is undefined

Reply:
One option I explored was to use <script setup lang="ts" generic="T"> instead of <script lang="ts" generic="T">. But I didn't invest much time into that